pyccel/codegen/printing/pycode.py

```diff
@@ -442,6 +442,10 @@ def _print_PythonList(self, expr):
         args = ', '.join(self._print(i) for i in expr.args)
         return '['+args+']'
 
+    def _print_PythonSet(self, expr):
+        args = ', '.join(self._print(i) for i in expr.args)
+        return '{'+args+'}'
```

Comment:
This will not work if you want to create an empty set. You will print `{}` but you need to print `set()` or `{,}`

Reply:
You're correct, I'll include a condition for that

Reply:
Has this been done now? Please provide the commit hash

Reply:
fixed : 5c0ec34